### ISSUE TITLE
Compilation error with Clang 10

### ISSUE THREAD

**Issue by ZaMaZaN4iK** [Stiffstream/sobjectizer#5]
Hi!
SObjectizer 5.7.0 cannot be compiled with Clang 10 (seems like the reason is in the updated `libstdc++`). I've got the following errors:
```
In file included from /home/zamazan4ik/.conan/data/sobjectizer/5.7.0/stiffstream/stable/build/ae1b6d0dfb9e84f72c2951a33321a99e6c04a334/sobjectizer/dev/so_5/disp/thread_pool/pub.cpp:13:
/home/zamazan4ik/.conan/data/sobjectizer/5.7.0/stiffstream/stable/build/ae1b6d0dfb9e84f72c2951a33321a99e6c04a334/sobjectizer/dev/so_5/disp/thread_pool/pub.hpp:427:13: error: no type named 'string_view' in namespace 'std'
        const std::string_view data_sources_name_base,
              ~~~~~^
/home/zamazan4ik/.conan/data/sobjectizer/5.7.0/stiffstream/stable/build/ae1b6d0dfb9e84f72c2951a33321a99e6c04a334/sobjectizer/dev/so_5/disp/thread_pool/pub.hpp:458:13: error: no type named 'string_view' in namespace 'std'
        const std::string_view data_sources_name_base,
              ~~~~~^
/home/zamazan4ik/.conan/data/sobjectizer/5.7.0/stiffstream/stable/build/ae1b6d0dfb9e84f72c2951a33321a99e6c04a334/sobjectizer/dev/so_5/disp/thread_pool/pub.hpp:491:37: error: no member named 'string_view' in namespace 'std'
                return make_dispatcher( env, std::string_view{}, thread_count );
                                             ~~~~~^
/home/zamazan4ik/.conan/data/sobjectizer/5.7.0/stiffstream/stable/build/ae1b6d0dfb9e84f72c2951a33321a99e6c04a334/sobjectizer/dev/so_5/disp/thread_pool/pub.hpp:524:10: error: no member named 'string_view' in namespace 'std'
                                std::string_view{},
                                ~~~~~^
In file included from /home/zamazan4ik/.conan/data/sobjectizer/5.7.0/stiffstream/stable/build/ae1b6d0dfb9e84f72c2951a33321a99e6c04a334/sobjectizer/dev/so_5/disp/adv_thread_pool/pub.cpp:12:
/home/zamazan4ik/.conan/data/sobjectizer/5.7.0/stiffstream/stable/build/ae1b6d0dfb9e84f72c2951a33321a99e6c04a334/sobjectizer/dev/so_5/disp/adv_thread_pool/pub.hpp:412:13: error: no type named 'string_view' in namespace 'std'
        const std::string_view data_sources_name_base,
              ~~~~~^
/home/zamazan4ik/.conan/data/sobjectizer/5.7.0/stiffstream/stable/build/ae1b6d0dfb9e84f72c2951a33321a99e6c04a334/sobjectizer/dev/so_5/disp/adv_thread_pool/pub.hpp:443:13: error: no type named 'string_view' in namespace 'std'
        const std::string_view data_sources_name_base,
              ~~~~~^
/home/zamazan4ik/.conan/data/sobjectizer/5.7.0/stiffstream/stable/build/ae1b6d0dfb9e84f72c2951a33321a99e6c04a334/sobjectizer/dev/so_5/disp/adv_thread_pool/pub.hpp:476:37: error: no member named 'string_view' in namespace 'std'
                return make_dispatcher( env, std::string_view{}, thread_count );
                                             ~~~~~^
3 errors generated.
```

As far as I see you just forgot to include `#include <string_view>` in `pub.hpp`

Hope it'll help.

**Comment by ZaMaZaN4iK** [comment]
Also will be fine, if you'll prepare a patch release with update corresponding packages for Conan/Vcpkg

**Comment by eao197** [comment]
I've pushed some changes to 5.7-dev branch. Could you check it on your machine?

**Comment by ZaMaZaN4iK** [comment]
Works fine. Thank you.

**Comment by eao197** [comment]
Thanks! I'll prepare updates for vcpkg and conan.

**Comment by eao197** [comment]
v.5.7.0.1 is available via conan now.

**Comment by ZaMaZaN4iK** [comment]
From my side all issues are solved by 5.7.0.1. So I think you can close the issue.

Thank you!
